script.js

```diff
@@ -1 +1,345 @@
+
+  // use the board data structure we passed in to create the correct size board
+  for (let i = 0; i < board.length; i += 1) {
+    // make a var for just this row of cards
+    const row = board[i];
+
+    // make an element for this row of cards
+    const rowElement = document.createElement('div');
+    rowElement.classList.add('row');
+
+    // make all the squares for this row
+    for (let j = 0; j < row.length; j += 1) {
+      // create the square element
+      const square = document.createElement('div');
+
+      // set a class for CSS purposes
+      square.classList.add('square');
+
+      // set the click event
+      // eslint-disable-next-line
+      square.addEventListener('click', (event) => {
+        // we will want to pass in the card element so
+        // that we can change how it looks on screen, i.e.,
+        // "turn the card over"
+        squareClick(event.currentTarget, i, j);
+      });
+
+      rowElement.appendChild(square);
+    }
+    boardElement.appendChild(rowElement);
+  }
+
+  return boardElement;
+};
+
+// username funtion, for user to input their username
+const userNameGame = () => {
+  // generate reset button
+  resetButton.innerHTML = 'Reset';
+  document.body.appendChild(resetButton);
+  resetButton.addEventListener('click', resetGame);
+
+  gameInfo.innerHTML = 'Let us know your name.';
+  document.body.appendChild(gameInfo);
+
+  // input field to put name
+  document.body.appendChild(field);
+
+  submitButton.innerHTML = 'Submit';
+  document.body.appendChild(submitButton);
+  submitButton.addEventListener('click', capture);
+};
+
+const initGame = () => {
+  // create this special deck by getting the doubled cards and
+  // making a smaller array that is ( boardSize squared ) number of cards
+  const doubleDeck = makeDeck();
+  const deckSubset = doubleDeck.slice(0, boardSize * boardSize);
+  deck = shuffleCards(deckSubset);
+
+  // deal the cards out to the board data structure
+  for (let i = 0; i < boardSize; i += 1) {
+    board.push([]);
+    for (let j = 0; j < boardSize; j += 1) {
+      board[i].push(deck.pop());
+    }
+  }
+  // remove the field and submit buttons
+  getElement('userInput').remove();
+  getElement('submit').remove();
+
+  // set game time
+  gameTime = 180;
+
+  // call timeout function
+  timeOut();
+
+  // generate timer message that counts down
+  timer.innerHTML = `⏱ You have ${minutes} mins ${seconds} secs to complete this game ⏱`;
+  document.body.appendChild(timer);
+
+  const boardEl = buildBoardElements(board);
+  boardEl.setAttribute('id', 'boardEl');
+  document.body.appendChild(boardEl);
+
+  gameInfo.innerHTML = `Hello 👋🏻 ${userName}. Click on a square to begin!`;
+  document.body.appendChild(gameInfo);
+
+  gameInfo2.innerHTML = '';
+  document.body.appendChild(gameInfo2);
+};
+
+userNameGame();
+
+// function to set the Timer
+const timeOut = () => {
+  const ref = setInterval(() => {
+    const minutes = Number(Math.floor(gameTime / 60));
```

Comment:
You can remove the const from lines 335 and 336 because you already create them as global variables in line 13 and 14. Then, just change const to let in line 13 and 14 to be able to reassign the values.